views/mixins/collapsible-sales.pug

```diff
@@ -0,0 +1,9 @@
+mixin collapsible-sales(category)
+    button(type="button" class="collapsible-button") #{category.category}
+    div(class="collapsible-content")
+        each item in category.items
+            a(href="/fewafewa")
```

Comment:
lol

Reply:
Hmm I am a bit confused as to what to do here. Are you referring to the inventory.json file?

Reply:
> lol

thought "fewafewa" was funny 😅 

> Would it be possible to remove the individual items from the /data-base file, and instead let that be configured each time the repo is created? 

I'm referring to the way we've populated this page by uploading all of the items and updating the `inventory.json` file accordingly. Ideally, we shouldn't have all of this live data committed to the repo, primarily because it's harder for the admins to manage (since images in the repo aren't currently handled by the image selector in the CMS), but also just out of good practice of not committing data we don't need to.

> Images would then live in the uploads folder (if you just copy your /inventory file structure into /uploads, that allow you to keep your setup locally with minimal changes).

I just mean to move the `/public/assets/img/inventory` folder to `/public/assets/img/uploads/inventory`! that will take the images out of this git commit, but still maintain the work you've done in setting up the file structure on your own computer.

(with the images no longer in the commit, we'd want to remove the associated items from the `/data-base/inventory.json` file in this commit as well)

does that clarify?

Reply:
Okay that makes sense.

I have also now realized that I should have merged the condensed navbar change earlier. So if there are too many conflicts I can revert to the previous navbar for now and redo it later on.